src/sklearn_evaluation/plot/regression.py

```diff
@@ -106,19 +134,77 @@ def prediction_error(y_true,
     r2 = model.score(y_true.reshape((-1, 1)), y_pred)
     plt.plot([], [], ' ', label=f"R2 = {round(r2,5)}")
 
-    _set_ax_settings(ax, 'y_true', 'y_pred', plot_name)
+    _set_ax_settings(ax, 'y_true', 'y_pred', 'Prediction Error')
     ax.legend(loc="upper left")
     return ax
 
-def _set_ax_settings(ax, xlabel, ylabel, title):
-    ax.set_title(title)
-    ax.set_xlabel(xlabel)
-    ax.set_ylabel(ylabel)
 
-def _check_parameter_validity(y_true, y_pred):
-    if any((val is None for val in (y_true, y_pred))):
-        raise ValueError('y_true and y_pred are needed to plot '
-                         'Residuals Plot')
+@SKLearnEvaluationLogger.log(feature='plot')
+def cooks_distance(
+    X,
+    y,
+    ax=None
+):
+    """Plots cooks distance.
+
+        Parameters
+        ----------
+        X : array-like, 2D
+            Training data
+            Refer https://numpy.org/doc/stable/glossary.html#term-array-like
+
+        y : array-like, 1D
+            Target data
+            Refer https://numpy.org/doc/stable/glossary.html#term-array-like
+
+        Returns
+        -------
+        ax: matplotlib Axes
+            Axes containing the plot
+
+        Examples
+        --------
+        .. plot:: ../../examples/cooks_distance.py
```

Comment:
i think this needs updates since we merged the jupyter book structure: `.. plot:: ../examples/cooks_distance.py`

Reply:
also, resolve file conflicts